libasn1fix/asn1fix_constr.c

```diff
@@ -60,7 +60,7 @@ asn1f_pull_components_of(arg_t *arg) {
 		coft = asn1p_expr_clone(terminal, 1 /* Skip extensions */);
 		if(!coft) return -1;	/* ENOMEM */
 
-		if(0) {
+		if(1) {
```

Comment:
Questionable. Will likely break something else, such as constraints, as indicated in the following comment.